cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -78,126 +80,144 @@ class CorrectIlluminationCalculate(cpm.Module):
+   (*{IC_REGULAR} + {SM_NONE} + {EA_EACH}*) because the illumination
+   function would be identical to the original image and applying it
+   will yield a blank image. You either need to smooth each image, or
+   you need to use *{EA_ALL}* images.
+-  *{IC_BACKGROUND}:* If you think that the background (dim points)
+   between objects show the same pattern of illumination as your objects
+   of interest, you can choose the *{IC_BACKGROUND}* method. Background
+   intensities finds the minimum pixel intensities in blocks across the
+   image (or group of images if you are in *{EA_ALL}* mode) and is most
+   often applied by subtraction using the **CorrectIlluminationApply**
+   module. *Note:* if you will be using the *{DOS_SUBTRACT}* option in
+   the **CorrectIlluminationApply** module, you almost certainly do not
+   want to rescale the illumination function.
+
+Please note that if a mask was applied to the input image, the pixels
+outside of the mask will be excluded from consideration. This is useful,
+for instance, in cases where you have masked out the well edge in an
+image from a multi-well plate; the dark well edge would distort the
+illumination correction function along the interior well edge. Masking
+the image beforehand solves this problem.
+""".format(**{
+                "IC_REGULAR": IC_REGULAR,
+                "EA_ALL": EA_ALL,
+                "EA_EACH": EA_EACH,
+                "SM_NONE": SM_NONE,
+                "IC_BACKGROUND": IC_BACKGROUND,
+                "DOS_SUBTRACT": DOS_SUBTRACT
+            })
+        )
 
         self.dilate_objects = cps.Binary(
-                "Dilate objects in the final averaged image?", False, doc='''
-            <i>(Used only if the Regular method is selected)</i><br>
-            For some applications, the incoming images are binary and each object
-            should be dilated with a Gaussian filter in the final averaged
-            (projection) image. This is for a sophisticated method of illumination
-            correction where model objects are produced.
-            Select <i>%(YES)s</i> to dilate objects for this approach.
-            ''' % globals())
+            "Dilate objects in the final averaged image?", False, doc="""\
+*(Used only if the Regular method is selected)*
+For some applications, the incoming images are binary and each object
+should be dilated with a Gaussian filter in the final averaged
+(projection) image. This is for a sophisticated method of illumination
+correction where model objects are produced. Select *%(YES)s* to dilate
+objects for this approach.
+""" % globals())
 
         self.object_dilation_radius = cps.Integer(
-                "Dilation radius", 1, 0, doc='''
-            <i>(Used only if the "%(IC_REGULAR)s" method and dilation is selected)</i><br>
-            This value should be roughly equal to the original radius of the objects''' % globals())
+            "Dilation radius", 1, 0, doc="""\
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.